### PR TITLE
fix(app-gen): fix lint issues in ts files

### PR DESCRIPTION
**Pull request by unseen1980**
https://github.com/SAP/open-ux-tools/issues/1000

Remove redundant commas fixes eslint issues.

### CODE REVIEW

File: .changeset/bright-garlics-know.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ux/fe-fpm-writer': patch
```
[the remark this answers is not in the document]
Comment (unseen1980) [reply .changeset/bright-garlics-know.md:2]: Should be ok now, thanks!

**Review by devinea — APPROVED** [review]
Thanks @unseen1980

**Review by 815are — APPROVED** [review]
Checked:
1. controller.ts templates are updated correctly;
2. snapshots updated correctly;
3. I did not find other templates with same issue;

I did not test with extensions

**Comment by unseen1980** [comment]
Thank you all

**Pull request merged**